frontend/integration-tests/tests/kubevirt/vm.nic.binding.scenario.ts

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-undef */
+import { $, browser } from 'protractor';
+
+import { appHost, testName } from '../../protractor.conf';
+import { isLoaded } from '../../views/crud.view';
+import { testNad, basicVmConfig, networkInterface, networkBindingMethod, networkWizardTabCol, networkTabCol } from './mocks';
+import * as vmView from '../../views/kubevirt/virtualMachine.view';
+import { removeLeakedResources, execCommandFromCli, assertUnorderListEqual, fillInput, selectDropdownOption, getDropdownOptions } from './utils';
+import Wizard from './models/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import * as wizardView from '../../views/kubevirt/wizard.view';
+
+describe('Test VM network interface binding method', () => {
+  const leakedResources = new Set<string>();
+  const wizard = new Wizard();
+  const vmName = `vm-${testName}`;
+  const vm = new VirtualMachine(vmName, testName);
+  const podNetworkList = [networkBindingMethod.pod, networkBindingMethod.default, networkBindingMethod.sriov];
+  const networkList = [networkBindingMethod.default, networkBindingMethod.sriov];
+
+  beforeAll(async() => {
+    execCommandFromCli(`echo '${JSON.stringify(testNad)}' | kubectl create -f -`);
```

Comment:
there are functions `createResources` and `deleteResources`, but the `deleteResources` cannot delete a vm which isn't derived from `getVmManifest`.

Reply:
```deleteResources``` deletes anything that has `name, namespace and kind` properties.
I'm a bit confused what is the point of execCommandFromCli method because it doesn't provide higher abstraction than execSync does, moreover it consumes the potential exception so if something happens, the test execution goes on as if nothing happened. I think we should get rid of it.

Reply:
good to know that these utility functions were created. Maybe we could use them here?

\+ having methods for single resource would be beneficial as well

Reply:
Updated `deleteResources` and use it now.